apps/ledger-live-desktop/src/renderer/modals/Receive/steps/StepAccount.tsx

```diff
@@ -128,7 +131,11 @@ export default function StepAccount({
           <Alert type="warning" learnMoreUrl={url} mt={3}>
             <Trans
               i18nKey={`receive.steps.chooseAccount.${
-                account.type === "TokenAccount" ? "verifyTokenType" : "warningTokenType"
+                account.type === "TokenAccount"
+                  ? startsWithVowel(tokenType)
+                    ? "verifyTokenType"
+                    : "verifyATokenType"
+                  : "warningTokenType"
```

Comment:
@hedi-edelbloute this fixes the warning message when obtaining the address for a token transaction. It uses "an" when the list of tokens start with a vowel, and "a" when not.

Here's the result:
<img width="470" alt="Screenshot 2025-04-03 at 10 11 19" src="https://github.com/user-attachments/assets/265561f0-dc63-4e30-b3f2-62bc73236215" />
<img width="466" alt="Screenshot 2025-04-03 at 10 11 05" src="https://github.com/user-attachments/assets/c61b32aa-c151-44fb-878a-baedb9d00c65" />

Reply:
@semeano We should just change the text, might be an issue on other languages. We can't edit code to match language issue

Please verify {asset} is of this type : {type}
Any other crypto asset...

Reply:
Maybe this should be addressed in a different PR/ticket. I'll rollback the changes